lib/web/mage/sticky.js

```diff
@@ -41,8 +44,34 @@ define([
 
             if (!isStatic && this.element.is(':visible')) {
                 offset = $(document).scrollTop() - this.parentOffset;
+
+                if (typeof this.options.spacingTop === 'function') {
+                    offset += this.options.spacingTop();
+                } else {
+                    offset += this.options.spacingTop;
+                }
+
                 offset = Math.max(0, Math.min(offset, this.maxOffset));
-                this.element.css('top', offset);
+
+                if (offset &&
+                    this.options.offsetTop &&
+                    !this.element.hasClass(this.options.stickyClass)) {
```

Comment:
The logic inside of this `if` should work for the non-sticked element, when `offsetTop` is used. (It makes smooth return of the items with `offsetTop` parameter, when scrolling up)

In other words, I check if an element should be stuck (offset > 0), `offsetTop` is used and an element is not yet stuck.